util/logutil/log.go

```diff
@@ -228,6 +228,17 @@ func WithCategory(ctx context.Context, category string) context.Context {
 	return WithFields(ctx, zap.String("category", category))
 }
 
+// WithTraceFields attaches trace fields to context
+func WithTraceFields(ctx context.Context, info *model.TraceInfo) context.Context {
+	if info == nil {
+		return WithFields(ctx)
+	}
+	return WithFields(ctx,
+		zap.Uint64("conn", info.ConnectionID),
+		zap.String("session_alias", info.SessionAlias),
```

Comment:
Do u think it's a good idea to defining 'session_alias" as a constant?

Reply:
Sure. I'll update the PR.